lti_consumer/templates/html/lti_1p3_studio.html

```diff
@@ -45,5 +45,19 @@
             <b>OIDC Callback URL: </b>
             ${oidc_callback}
         </p>
+        <p>
+            <b>Test LTI Launch: </b>
+
+            <a
```

Comment:
Can you make sure the button displays as the mockup?

![image](https://user-images.githubusercontent.com/5691347/99539147-a2e4dc80-298c-11eb-970f-26940219b8db.png)

Reply:
@viadanna, Used a button instead of a link now. The style should be ok.